example/android/build.gradle

```diff
@@ -31,9 +40,14 @@ allprojects {
             // Android JSC is installed from npm
             url("$rootDir/../node_modules/jsc-android/dist")
         }
-
+        mavenCentral {
+            // We don't want to fetch react-native from Maven Central as there are
+            // older versions over there.
+            content {
+                excludeGroup "com.facebook.react"
+            }
+        }
         google()
-        mavenCentral()
         maven { url 'https://www.jitpack.io' }
```

Comment:
Came to see if this was working but seeing same problem as us, we had to add

`maven {
            url "https://pdftron-maven.s3.amazonaws.com/snapshot"
        }`

here otherwise at build it can't resolve PDFTron as a dependency.

Reply:
Here's a sample project where I ran:
- `npx react-native init`
- `yarn add github:PDFTron/pdftron-react-native`
- `yarn android`

and get the same error 

https://github.com/catenda/pdftron-example-react-native/tree/9faff8bdebbd439ae3d0f01c32236b3a8ce3c340

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:processDebugResources'.
> Could not resolve all task dependencies for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.pdftron:pdftron:9.3.0-beta02.
     Searched in the following locations:
       - file:[REDACTED]/pdftron-example-react-native/node_modules/react-native/android/com/pdftron/pdftron/9.3.0-beta02/pdftron-9.3.0-beta02.pom
       - file:[REDACTED]/pdftron-example-react-native/node_modules/jsc-android/dist/com/pdftron/pdftron/9.3.0-beta02/pdftron-9.3.0-beta02.pom
       - https://repo.maven.apache.org/maven2/com/pdftron/pdftron/9.3.0-beta02/pdftron-9.3.0-beta02.pom
       - https://dl.google.com/dl/android/maven2/com/pdftron/pdftron/9.3.0-beta02/pdftron-9.3.0-beta02.pom
       - https://www.jitpack.io/com/pdftron/pdftron/9.3.0-beta02/pdftron-9.3.0-beta02.pom
     Required by:
         project :app > project :react-native-pdftron
   > Could not find com.pdftron:tools:9.3.0-beta02.
     Searched in the following locations:
       - file:[REDACTED]/pdftron-example-react-native/node_modules/react-native/android/com/pdftron/tools/9.3.0-beta02/tools-9.3.0-beta02.pom
       - file:[REDACTED]/pdftron-example-react-native/node_modules/jsc-android/dist/com/pdftron/tools/9.3.0-beta02/tools-9.3.0-beta02.pom
       - https://repo.maven.apache.org/maven2/com/pdftron/tools/9.3.0-beta02/tools-9.3.0-beta02.pom
       - https://dl.google.com/dl/android/maven2/com/pdftron/tools/9.3.0-beta02/tools-9.3.0-beta02.pom
       - https://www.jitpack.io/com/pdftron/tools/9.3.0-beta02/tools-9.3.0-beta02.pom
     Required by:
         project :app > project :react-native-pdftron
   > Could not find com.pdftron:collab:9.3.0-beta02.
     Searched in the following locations:
       - file:[REDACTED]/pdftron-example-react-native/node_modules/react-native/android/com/pdftron/collab/9.3.0-beta02/collab-9.3.0-beta02.pom
       - file:[REDACTED]/pdftron-example-react-native/node_modules/jsc-android/dist/com/pdftron/collab/9.3.0-beta02/collab-9.3.0-beta02.pom
       - https://repo.maven.apache.org/maven2/com/pdftron/collab/9.3.0-beta02/collab-9.3.0-beta02.pom
       - https://dl.google.com/dl/android/maven2/com/pdftron/collab/9.3.0-beta02/collab-9.3.0-beta02.pom
       - https://www.jitpack.io/com/pdftron/collab/9.3.0-beta02/collab-9.3.0-beta02.pom
     Required by:
         project :app > project :react-native-pdftron

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s
``` 

Here's the output of `npx react-native info` in case it can shed any light.

```
info Fetching system and libraries information...
System:
    OS: macOS 12.5
    CPU: (20) arm64 Apple M1 Ultra
    Memory: 25.79 GB / 64.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.15.0 - ~/Library/Caches/fnm_multishells/6888_1658998868169/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 8.5.5 - ~/Library/Caches/fnm_multishells/6888_1658998868169/bin/npm
    Watchman: 2022.07.04.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.15 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.0.0 => 18.0.0
    react-native: 0.69.3 => 0.69.3
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Reply:
Unfortunately we still cannot seem to reproduce the issue. We've tried cloning and building the sample project on an M1 Macbook as well (as indicated by your info output), but there were still no issues building and running the Android app. 

We'll keep an eye on this issue though and see if any other users encounter it.

Reply:
Thanks for taking the time to look into this. I've been able to confirm on my side that this is an environment issue. This can be marked as resolved.